### PR TITLE
Milestone 1: mobile version skeleton of the portfolio 

### PR DESCRIPTION
**Pull request by rachidelaid**
- added a section for the projects with the class "works" with display grid.
- each project has a class "work-card" with display flex and direction column, and just 2 direct children (img & .info).
- added a section for the about my self with a class "about" with display grid, and just 2 direct children (.info & .list)
- the first child with the class "info" is set to display flex.
- the second child with the class "list" has 3 `div` elements for the dropdowns and a `ul` for the first `div` because it has content.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -79,6 +81,118 @@ p {
...
+
+.subTitle li:first-child {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  list-style-type: none;
+}
+
+button {
```
Comment (keneogu) [inline style.css:158]: - [x] I suggest you follow the button styling from the Figma design. The button should have styling when active, hover or pressed.

File: style.css [hunk 2]
```diff
@@ -79,6 +81,118 @@ p {
   list-style-type: none;
 }
 
-.socials li:not(:last-child) {
+.socials img {
   margin-right: 20px;
 }
+
+.works {
```
Comment (keneogu) [inline style.css:90]: - [ ] There is no margin on the works section. i suggest you follow the design styling. Add margins (top, bottom, left, right) to the works section.

File: style.css [hunk 3]
```diff
@@ -79,6 +81,118 @@ p {
...
+  margin: 12px 0 24px 0;
+  display: flex;
+  list-style-type: none;
+  color: #6070ff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.langauges li {
```
Comment (keneogu) [inline style.css:143]: - [ ] From the design, the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") has a background color please add it.
Your design
pro

Figma design
pro1

File: index.html [hunk 4]
```diff
@@ -41,5 +41,161 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+    <section class="about">
+      <div class="info">
+        <h1>About Myself</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <a href="#">LET’S CONNECT</a>
```
Comment (keneogu) [inline index.html]: - [ ] Let's connect is a title, always make use of the header `<hx>` tag.

File: index.html [hunk 5]
```diff
@@ -41,5 +41,161 @@ <h1>I’m Mariane Glad to see you!</h1>
...
+          </li>
+          <li>
+            <img src="./assets/peace.svg" alt="peace icon" />
+          </li>
+          <li>
+            <img src="./assets/twitter.svg" alt="twitter icon" />
+          </li>
+          <li>
+            <img src="./assets/meduim.svg" alt="medium icon" />
+          </li>
```
Comment (keneogu) [inline index.html]: - [ ] Each icon should be a link. this makes it easy for companies or recruiters to connect with you

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @rachidelaid , 

Good job so far!

- Linters working properly
- Well documented PR description
- Commit messages are meaningful
- Good HTML structure
- Good UI design

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: style.css (hunk 2 above)
Comment (rachidelaid) [reply style.css:90]: works doesn't have any margin, it's just padding like in Figma.
can you take another look please and tell if I have to remove the padding-top

image

**Review by elyor-doniyorov — APPROVED** [review]
Hi @rachidelaid , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**